### PR TITLE
ADEN-5469 Prepare News&Stories tests for f2

### PR DESCRIPTION
**Pull request by nandy-andy**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [FandomUrlBuilder.java#L38](https://github.com/Wikia/selenium-tests/blob/f00812b664602971ed06ba5a4d8aefa5be3be1e3/src/test/java/com/wikia/webdriver/common/core/url/FandomUrlBuilder.java#L38): Define a constant instead of duplicating this literal "http://" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/common/templates/fandom/AdsFandomTestTemplate.java [hunk 1]
```diff
@@ -14,15 +17,24 @@ protected void loadFirstPage() {
...
-    switch (pageType) {
-      case PAGE_TYPE_HUB:
-        pageUrl = urlBuilder.getUrlForFandomHub(pageName);
-        break;
-      case PAGE_TYPE_ARTICLE:
-      default:
-        pageUrl = urlBuilder.getUrlForFandomPage(pageName);
-        break;
+    String queryString = Configuration.getQS();
+    Boolean isF2 = queryString.matches(".*f2(?=\\=|&|$|\\n).*");
```
Comment (dianafa) [inline src/test/java/com/wikia/webdriver/common/templates/fandom/AdsFandomTestTemplate.java]: I bet you know what would be the most evil request in the world here 😈

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [FandomUrlBuilder.java#L38](https://github.com/Wikia/selenium-tests/blob/ad2d631677682cb712d856f64fea4158383963a4/src/test/java/com/wikia/webdriver/common/core/url/FandomUrlBuilder.java#L38): Define a constant instead of duplicating this literal "http://" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [FandomUrlBuilder.java#L38](https://github.com/Wikia/selenium-tests/blob/bb4dec2280e7ed39e3b216e8fa21532691951a19/src/test/java/com/wikia/webdriver/common/core/url/FandomUrlBuilder.java#L38): Define a constant instead of duplicating this literal "http://" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [FandomUrlBuilder.java#L38](https://github.com/Wikia/selenium-tests/blob/958cea4196e8f7c744cd2708f275b929d175f57c/src/test/java/com/wikia/webdriver/common/core/url/FandomUrlBuilder.java#L38): Define a constant instead of duplicating this literal "http://" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request closed without merging**